classes/mail/Mail.inc.php

```diff
@@ -473,6 +473,11 @@ function send() {
 			$mailer->Host = Config::getVar('email', 'smtp_server');
 			$mailer->Username = Config::getVar('email', 'smtp_username');
 			$mailer->Password = Config::getVar('email', 'smtp_password');
+			if (Config::getVar('debug', 'show_stacktrace')) {
+				// debug level 3 represents client and server interaction, plus initial connection debugging
```

Comment:
Weird that they seem to have a mix of magic numbers and magic strings... I wish they'd use `define` like certain other projects ;)